modules/roles_profiles/manifests/profiles/mac_v3_signing.pp

```diff
@@ -31,16 +31,52 @@
 
             # we can add worker setup here like in gecko_t_osx_1014_generic_worker.pp
 
+            class { 'roles_profiles::profiles::logging':
```

Comment:
Should they not? I was following puppet's documentation. I can rework it to follow local style guides

Reply:
I wouldn't rework it since the logging profile is used in  a lot of other roles.  It is on my list of things to fix and shouldn't be addressed in this scope of work.